plugin_runner/plugin_runner.py

```diff
@@ -96,7 +97,9 @@ async def HandleEvent(self, request: Event, context: Any) -> EventResponse:
                     delta=compute_duration,
                 )
             except Exception as e:
-                log.error(traceback.format_exception(e))
+                for error_line_with_newlines in traceback.format_exception(e):
+                    for error_line in error_line_with_newlines.split("\n"):
+                        log.error(error_line)
```

Comment:
Makes exceptions caused by plugin code much more readable

Reply:
Does this remove the guarantee that the lines in the traceback will appear in order when they are slurped into ES? I feel like I've had that issue when looking at home-app tracebacks, but I'm not sure if I'm remembering correctly.

Reply:
I wouldn't think they'd get out of order. When you've seen this in the past, was it multiple containers or processes having their log output get interleaved?

Reply:
I have a memory of seeing log lines out of order, but you know how memory is.

ES will receive log lines with a timestamp from the original system, but if those time stamps are the same down to the millisecond/nanosecond, which I have seen in the case of tracebacks, then what happens? The lines may not necessarily be received by ES in order.

We'll have to see how this shakes out, but I'll just throw it out there that having the traceback all on one line can be advantageous in some scenarios (it helps with the intereleaving thing too). If this turns out to be the case, maybe log both?

Reply:
We'll see how ES treats it, in the meantime I'll prioritize developer experience. This change is primarily to make it easier for plugin developers to debug their plugins, and they won't have access to our kibana.